tinker-sample-android/app/build.gradle

```diff
@@ -86,6 +86,7 @@ android {
          * you can use multiDex and install it in your ApplicationLifeCycle implement
          */
         multiDexEnabled true
+        multiDexKeepProguard file("tinker_multidexkeep.pro")
```

Comment:
I found in testing, that without this proguard file (which is recommended by tinker anyway), sometimes the Tinker code would move out of the first dex and cause the patch generation to fail.  Adding this made testing the patch consistent.